File: Day47/sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        n = len(nums)
        for i in range(n-1):
            m = i
            for j in range(i+1,n):
                if nums[m] > nums[j]:
                    m = j
            if nums[m] == 2:
                break
            nums[i],nums[m] = nums[m],nums[i]

File: Day99/sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {

        int count[3]{};
        for(int n : nums){
            count[n]++;
        }
        int i = 0;
        for(int k = 0; k < 3;k++){
            for(int j = 0; j < count[k];j++){
                nums[i+j] = k;
            }
            i += count[k];
        }
    }
};
